<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#menu_widget, #toolBox {
	background-color: #3333FF;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>0</number>
      </property>
      <widget class="QWidget" name="menu_widget" native="true">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #06162d;
color: #fff;
border: none;</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QToolBox" name="toolBox">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#toolBox {
	color: #fff;
}

#toolBox::tab {
	padding-left:5px;
	text-align:left;
	border-radius: 2px;
}


#toolBox::tab:selected {
	background-color: #2d9cdb;
	font-weight: bold;
}

#toolBox QPushButton {
	padding:5px 0px 5px 20px;
	text-align:left;
	border-radius: 3px;
}

#toolBox QPushButton:hover {
	background-color: #85C1E9;
}

#toolBox QPushButton:checked {
	background-color: #3498DB;
}

</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="tabSpacing">
           <number>10</number>
          </property>
          <widget class="QWidget" name="general_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>161</width>
             <height>280</height>
            </rect>
           </property>
           <attribute name="icon">
            <iconset resource="PyQt5-Video-Book-main_3/PyQt5-Video-Book-main/#008_app_structure_v2/static/resource.qrc">
             <normaloff>:/icon/icon/home-4-48 (2).ico</normaloff>:/icon/icon/home-4-48 (2).ico</iconset>
           </attribute>
           <attribute name="label">
            <string>STUDENTS</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>Recieve Payment</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>Transactions</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>Student Accounts</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Add Student</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_9">
              <property name="text">
               <string>Dash Board</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="cars_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>161</width>
             <height>280</height>
            </rect>
           </property>
           <attribute name="icon">
            <iconset resource="PyQt5-Video-Book-main_3/PyQt5-Video-Book-main/#008_app_structure_v2/static/resource.qrc">
             <normaloff>:/icon/icon/car-4-48.ico</normaloff>:/icon/icon/car-4-48.ico</iconset>
           </attribute>
           <attribute name="label">
            <string>EDUCATORS</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>Add Educator</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_14">
              <property name="text">
               <string>Pay Educator</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_12">
              <property name="text">
               <string>Transactions</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>Dashboard</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="social_media_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>161</width>
             <height>280</height>
            </rect>
           </property>
           <attribute name="icon">
            <iconset resource="PyQt5-Video-Book-main_3/PyQt5-Video-Book-main/#008_app_structure_v2/static/resource.qrc">
             <normaloff>:/icon/icon/group-48.ico</normaloff>:/icon/icon/group-48.ico</iconset>
           </attribute>
           <attribute name="label">
            <string>SETTINGS</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_11">
              <property name="text">
               <string>Dashboard</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>388</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="main_widget" native="true">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Filter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_4"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_13">
                <property name="text">
                 <string>Search</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableWidget" name="tableWidget">
              <column>
               <property name="text">
                <string>Date</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Transaction #</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Learners Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Amount Paid</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <widget class="QWidget" name="layoutWidget_4">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>60</y>
              <width>217</width>
              <height>204</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="5" column="0">
              <layout class="QFormLayout" name="formLayout_9">
               <item row="0" column="0">
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>Address</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_15"/>
               </item>
              </layout>
             </item>
             <item row="6" column="0">
              <layout class="QFormLayout" name="formLayout_10">
               <item row="0" column="0">
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>Email Address</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_16"/>
               </item>
              </layout>
             </item>
             <item row="4" column="0">
              <layout class="QFormLayout" name="formLayout_11">
               <item row="0" column="0">
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>Parent's Cell:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_17"/>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_20">
                 <property name="text">
                  <string>Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_18"/>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <layout class="QFormLayout" name="formLayout_12">
               <item row="0" column="0">
                <widget class="QLabel" name="label_21">
                 <property name="text">
                  <string>Date of Birth</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QDateEdit" name="dateEdit_2"/>
               </item>
              </layout>
             </item>
             <item row="0" column="0">
              <layout class="QFormLayout" name="formLayout_13">
               <item row="0" column="0">
                <widget class="QLabel" name="label_22">
                 <property name="text">
                  <string>Surname</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_19">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="0">
              <layout class="QFormLayout" name="formLayout_14">
               <item row="0" column="0">
                <widget class="QLabel" name="label_23">
                 <property name="text">
                  <string>Gender</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="comboBox_4"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget_5">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>270</y>
              <width>123</width>
              <height>26</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <spacer name="horizontalSpacer_11">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_18">
               <property name="text">
                <string>Submit</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget_6">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>10</y>
              <width>235</width>
              <height>29</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <spacer name="horizontalSpacer_12">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_24">
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>ADD STUDENT</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_13">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="page_3">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <spacer name="horizontalSpacer_14">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_20"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_19">
                <property name="text">
                 <string>Search</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTabWidget" name="tabWidget_2">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab_3">
               <attribute name="title">
                <string>Financial details</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QTableWidget" name="financial_details_tableWidget_2">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <column>
                   <property name="text">
                    <string>Date</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Transaction #</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Description</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Amount Paid</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Amount Owing</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_4">
               <attribute name="title">
                <string>Biographical Details</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_4">
           <widget class="QWidget" name="layoutWidget_3">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>80</y>
              <width>291</width>
              <height>121</height>
             </rect>
            </property>
            <layout class="QFormLayout" name="formLayout_8">
             <item row="0" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_12"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Surname</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_13"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Amount</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_14"/>
             </item>
             <item row="3" column="0" colspan="2">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <spacer name="horizontalSpacer_10">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_17">
                 <property name="text">
                  <string>Pay</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="PyQt5-Video-Book-main_3/PyQt5-Video-Book-main/#008_app_structure_v2/static/resource.qrc"/>
  <include location="PyQt5-Video-Book-main_3/PyQt5-Video-Book-main/#008_app_structure_v2/static/resource.qrc"/>
 </resources>
 <connections/>
</ui>
